package.json

```diff
@@ -50,12 +50,20 @@
     "eslint-plugin-jest": "21.2.0",
     "eslint-plugin-react": "7.4.0",
     "eslint-plugin-react-native": "3.1.0",
+    "husky": "^0.14.3",
+    "lint-staged": "^6.0.0",
     "npm-run-all": "^4.1.1",
     "prettier": "^1.9.2"
   },
   "peerDependencies": {
     "babel-eslint": ">=7.2.3",
     "eslint": ">=4.9.0",
     "eslint-plugin-import": ">=2.8.0"
+  },
+  "lint-staged": {
+    "gitDir": "../",
```

Comment:
Is this line right?  It's been a while, but IIRC, we only need it at all if our git directory isn't the current directory.

Reply:
We actually don't need this anymore and it was removed.